wtr-utils.js

```diff
@@ -247,6 +247,7 @@ const createUnitTestsConfig = (config) => {
       config: {
         ui: 'bdd',
         timeout: '10000',
+        retries: 2,
```

Comment:
I assume this will retry the tests both on CI and locally. I wonder if we might not need it locally.

Reply:
Not sure if a param can be passed from the shell to web-test-runner, so the `retries` can be set. Not sure if it's a problem to retry it locally as well.

Reply:
Could be changed to `retries: process.env.GITHUB_REF ? 2 : 0`

Reply:
BTW, 2 here means it will try running failed tests 2 more times, so 3 times in total. Is it ok, or one more time is enough? WDYT?

Reply:
Added the condition to retry only in GH